flowey/flowey_cli/src/pipeline_resolver/ado_yaml.rs

```diff
@@ -319,13 +307,31 @@ EOF
             )?;
         }
 
+        let mut flowey_executable_bash = String::new();
```

Comment:
you've got some funky format! inside writeln! stuff going on here. I think you just need a single `let flowey_executable_bash = format!()` line (with maybe a trailing `.trim_start().to_string()`)